recipes/opencolorio/all/conanfile.py

```diff
@@ -33,9 +40,12 @@ def requirements(self):
         self.requires("lcms/2.11")
         self.requires("yaml-cpp/0.6.3")
 
+    def validate(self):
+        if self.settings.arch not in ["x86", "x86_64"]:
+            raise ConanInvalidConfiguration("Only x86/x86_64 supported")
```

Comment:
@jgsogo I think it's not true

OpenColorio has an option `OCIO_USE_SSE`: https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/c835f319247099fabb622680af5d8416dae8dded/CMakeLists.txt#L129
When turned OFF, `emmintrin.h` is not included: https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/0021372b4dba3444e21a36d846b327efad054f8d/src/OpenColorIO/SSE.h#L9-L12